singer_encodings/csv_helper.py

```diff
@@ -117,7 +117,11 @@ def get_row_iterator(self, file_stream, delimiter, headers_in_catalog = None):
 
         # Replace any NULL bytes in the line given to the Reader
         reader = csv.reader((line.replace('\0', '') for line in file_stream), delimiter=delimiter)
-        self.all_csv_headers = next(reader)
+        try:
+            self.all_csv_headers = next(reader)
+        except StopIteration:
+            # Return None if CSV file is empty.
+            return None
```

Comment:
@dbshah1212 I am dubious for one scenario where a CSV file has 10 rows and the last row is empty. Can you please let me know the execution of this scenario?

Reply:
It breaks only if it won't get anything in the file. For the case 10 rows and the last row is empty, tap-won't break with or without this change and we won't get any field in the last record. So the last record will be something like below: 

`{"type": "RECORD", "stream": "Test", "record": {"_sdc_source_bucket": "bucket", "_sdc_source_file": "test_db/test_abc.csv", "_sdc_source_lineno": 11}}`